test/suites/ACID.m

```diff
@@ -1818,10 +1818,10 @@
 function [stat] = parameterCurve3d()
   stat.description = 'Parameter curve in 3D with text boxes in-/outise axis.';
   stat.issues = 378;
-
-  ezplot3('sin(t)','cos(t)','t',[0,6*pi]);
+  stat.issues = 790;
```

Comment:
Mlease merge this line with `stat.issues = 378;` above, so it reads `stat.issues = [378, 790]` -- otherwise it overwrites the previous value.